tests/schemas/test_invalid_date_range_period.json

```diff
@@ -7,8 +7,11 @@
     "description": "A test schema for different date formats",
     "theme": "default",
     "legal_basis": "StatisticsOfTradeAct",
-    "variables": {
-        "period": "{{ format_conditional_date (answers.period_from, exercise.start_date)}} to {{ format_conditional_date (answers.period_to, exercise.end_date)}}"
+    "metadata": {
+        "trad_as": "string",
+        "user_id": "string",
+        "period_id": "string",
+        "ru_name": "string"
```

Comment:
Am I correct in thinking that `"string"` above is defining the expected type of the metadata and not a dummy metadata value?

Reply:
@samiwel The keys are to define the type of validator runner will use for each type of metadata. For dates it will be `"ref_p_start_date": "date"`. Its not so much the `type` but what the data represents. I can explain it directly if its not clear.

Reply:
Okay, I understand. The only comment I have about that is that Author requires specific knowledge about the various types of validators supported by Survey Runner. For now it's fine, we can manage with strings and dates, but longer term it might be nice to explicitly define the available validators in the schema definitions.

e.g.

```
"trad_as": {
  "validator": "string"
}
```

then you can define `validator` to be an enum type of "string", "date", ...

Reply:
FYI: Remove `ru_name` and `trad_as` as it is not just implemented into runner directly.

@nealedj What do you think about Sam's comment. Is it worth changing now if it is need by runner in the future?. It would require runner changes with schema and logic but I think if its going to change, why not implement it properly in the first place?

Reply:
Yep that works.

If you're making other changes already then this seems like a good way to go. It would also allow us to add further keys to metadata if we needed to for future requirements (I.e. `optional`)